apps/etterlatte-behandling/src/main/kotlin/vilkaarsvurdering/ektedao/VilkaarsvurderingRepository.kt

```diff
@@ -0,0 +1,447 @@
+package no.nav.etterlatte.vilkaarsvurdering.ektedao
+
+import kotliquery.Row
+import kotliquery.Session
+import kotliquery.queryOf
+import no.nav.etterlatte.common.ConnectionAutoclosing
+import no.nav.etterlatte.libs.common.sak.SakId
+import no.nav.etterlatte.libs.common.tidspunkt.toLocalDatetimeUTC
+import no.nav.etterlatte.libs.common.tidspunkt.toTidspunkt
+import no.nav.etterlatte.libs.common.tidspunkt.toTimestamp
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.Delvilkaar
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.Vilkaar
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.VilkaarVurderingData
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.Vilkaarsvurdering
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.VilkaarsvurderingResultat
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.VilkaarsvurderingUtfall
+import no.nav.etterlatte.libs.common.vilkaarsvurdering.VurdertVilkaar
+import no.nav.etterlatte.libs.database.tidspunkt
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+import java.time.YearMonth
+import java.util.UUID
+
+fun <T> Session.hent(
+    queryString: String,
+    params: Map<String, Any> = mapOf(),
+    converter: (r: Row) -> T,
+) = queryOf(statement = queryString, paramMap = params)
+    .let { query -> this.run(query.map { row -> converter.invoke(row) }.asSingle) }
+
+fun Session.oppdater(
+    query: String,
+    params: Map<String, Any?>,
+    ekstra: ((tx: Session) -> Unit)? = null,
+) = queryOf(statement = query, paramMap = params)
+    .let { this.run(it.asUpdate) }
+    .also { ekstra?.invoke(this) }
+
+class VilkaarsvurderingRepository(
+    private val connectionAutoclosing: ConnectionAutoclosing,
+    private val delvilkaarRepository: DelvilkaarRepository,
+) {
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    fun hent(behandlingId: UUID): Vilkaarsvurdering? =
```

Comment:
Kanskje utanfor scope for denne PR-en, men denne metoden synes eg er eit erketypisk eksempel på korfor eg liker `hent`-hjelpemetodar som den definert over. Veldig mykje seremoni, og krevjande å sjå kva som er logikken her.